src/dos/drive_virtual.cpp

```diff
@@ -348,11 +348,12 @@ Virtual_File::Virtual_File(const vfile_data_t& in_data)
 
 bool Virtual_File::Read(uint8_t * data,uint16_t * size) {
 	uint32_t left = file_data->size() - file_pos;
+	uint8_t* src  = file_data->data() + file_pos;
```

Comment:
Nice :smile:. Add an assert for verification if `file_pos` is small enough to not not trigger out of bounds memory access. I guess it does not, but if someone will change code around then we want this to blow up as soon as possible.